CMakeLists.txt

```diff
@@ -15,44 +15,22 @@
 cmake_minimum_required(VERSION 3.4.3)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
+option(ONEDPL_FPGA_STATIC_REPORT "Enable the static report generation for the FPGA device" OFF)
+option(ONEDPL_USE_AOT_COMPILATION "Enable the ahead of time compilation via OCLOC compiler" OFF)
```

Comment:
Can you be more specific about what is not relevant after that commit?
I rebased, but I don't think the `options` themselves were modified in #794. 
I've merely moved them above the `project()` call, so that they stay together as a group.  I need the newly added option to be available before the `project()` call.